### DISCUSSION TITLE
Add all channels from https://photocall.tv/.

### DISCUSSION THREAD

**Discussion by heroku-miraheze** [iptv-org/community#244] [Show and tell]
One idea, we need to add all channels from https://photocall.tv/, but the streams URL will be based on Base64 decoded in str.
Thank you.

**Comment by jamie-203** [comment]
Very interested ----can u explain how to decode manually the few streams I need?
Many thanx

**Reply by heroku-miraheze** [reply]
Copying base64 encoded in str code, and decoding it in https://base64decode.org.

**Reply by jamie-203** [reply]
Thanx will give it a try

**Comment by BellezaEmporium** [comment]
It seems that those are official streams on their majority, could give it a try. However some are tokened and it's the PHP page itself doing the trick for getting the token, some might go under my iptv-all service.
